### PR TITLE
[Storage Version Migrator] docs: adds feature gates and runtime config information

### PR DESCRIPTION
**Pull request by nilekhc**
Adds feature gates and runtime config information required to use storage version migration feature.

Fixes https://github.com/kubernetes/website/issues/46862

/cc @enj @jpbetz @deads2k

### CODE REVIEW

File: content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md [hunk 1]
```diff
@@ -26,6 +26,12 @@ Install [`kubectl`](/docs/tasks/tools/#kubectl).
 
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
+Set `StorageVersionMigrator` and `InformerResourceVersion` feature gates to
+`true`.
+
+Enable storage version migration REST api by setting runtime config
+`storagemigration.k8s.io/v1alpha1` to `true`.
+
```
Comment (nilekhc) [inline content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md:34]: @sftim Is there a better way to add feature gates and runtime config info? Any example I can follow?

Comment (sftim) [reply content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md:34]: There's a guide at https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates but it doesn't cover specifically what you're asking.

File: content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md [hunk 2]
```diff
@@ -26,6 +26,12 @@ Install [`kubectl`](/docs/tasks/tools/#kubectl).
 
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
+Set `StorageVersionMigrator` and `InformerResourceVersion` feature gates to
+`true`.
```
Comment (sftim) [inline content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md:30]:
```suggestion
Ensure that your cluster has the `StorageVersionMigrator` and `InformerResourceVersion`
[feature gates](/docs/reference/command-line-tools-reference/feature-gates/)
enabled. You will need control plane administrator access to make that change.
```

(You typically don't set them to true, you add them to a list of enabled gates)

File: content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md [hunk 3]
```diff
@@ -26,6 +26,12 @@ Install [`kubectl`](/docs/tasks/tools/#kubectl).
...
+
+Enable storage version migration REST api by setting runtime config
+`storagemigration.k8s.io/v1alpha1` to `true`.
```
Comment (sftim) [inline content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md:33]:
```suggestion
`storagemigration.k8s.io/v1alpha1` to `true` for the API server. For more information on
how to do that,
read [enable or disable a Kubernetes API](/docs/tasks/administer-cluster/enable-disable-api/).
```

**Review by sftim — COMMENTED** [review]
Try these changes?

**Comment by enj** [comment]
LGTM once Tim's comments have been addressed.

**Comment by nilekhc** [comment]
> Try these changes?

@sftim PTAL when you get a chance. Thanks for review!

**Pull request merged**